mkversion.sh

```diff
@@ -1,18 +1,37 @@
 #!/bin/sh
-set -ue
-cd "$(dirname "$0")"
+set -e
+
+# debugging if anything fails is tricky as dh-golang eats up all output
+# uncomment the lines below to get a useful trace if you have to touch
+# this again (my advice is: DON'T)
+#set -x
+#logfile=/tmp/mkversions.log
+#exec >> $logfile 2>&1
+#echo "env: $(set)"
+#echo "mkversion.sh run from: $0"
+#echo "pwd: $(pwd)"
+
+
+# run from go-generate inside "cmd", in this case we need to go to the
+# parent dir
+if [ "$GOPACKAGE" = "cmd" ]; then
```

Comment:
Can you tell if that directory is different from `$(dirname $0)`, what was the root cause of the bug?